jetty-util/src/test/java/org/eclipse/jetty/util/PoolTest.java

```diff
@@ -506,10 +532,10 @@ public void testMultiplexMaxUsageReachedAcquireThenReleaseThenRemove(int cacheSi
     }
 
     @ParameterizedTest
-    @MethodSource(value = "cacheSize")
-    public void testUsageCountAfterReachingMaxMultiplexLimit(int cacheSize)
+    @MethodSource(value = "strategy")
+    public void testUsageCountAfterReachingMaxMultiplexLimit(Factory factory)
     {
-        Pool<String> pool = new Pool<>(1, cacheSize);
+        Pool<String> pool = factory.getPool(1);
         pool.setMaxMultiplex(2);
         pool.setMaxUsageCount(10);
         pool.reserve(-1).enable("aaa", false);
```

Comment:
This test contains an unused variable `e2` that I would use in asserts, at least to verify that `e2 == e1`.